builtin/providers/aws/resource_aws_s3_bucket_object.go

```diff
@@ -26,6 +25,36 @@ func resourceAwsS3BucketObject() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"cache_control": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"content_disposition": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"content_encoding": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"content_language": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"content_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Seems like at least this one needs to be `computed` as well, I'm getting a plan loop with this config: 

```hcl
provider "aws" {
  region = "us-west-2"
}

resource "aws_s3_bucket" "object_bucket" {
	bucket = "tf-object-test-bucket-1234"
}

resource "aws_s3_bucket_object" "object" {
	bucket = "${aws_s3_bucket.object_bucket.bucket}"
	key = "test-key"
	source = "terraform.tfstate"
}
```

```console
~ aws_s3_bucket_object.object
    content_type: "binary/octet-stream" => ""
```